### PR TITLE
import user index to fix bug

### PR DESCRIPTION
**Pull request by Joeper214**
ref: https://skyarch.backlog.jp/view/NC-583.

Please review and merge PR. @root-beer .

Also, thank you for reporting this issue. :bow:

### CODE REVIEW

File: frontend/src/users_admin.ts [hunk 1]
```diff
@@ -5,11 +5,12 @@
 //
 // http://opensource.org/licenses/mit-license.php
 //
-//= require user_index
+
 
 /// <reference path="../declares.d.ts" />
 
 import {Confirm, Alert, AlertForAjaxStdError} from './modal';
+import 'user_index'
```
Comment (skyarch-networks) [inline frontend/src/users_admin.ts]: __[tslint]__
* missing semicolon

**Comment by Joeper214** [comment]
But test is failed. :scream:

**Comment by root-beer** [comment]
I tried but it seems to be not working...

**Comment by Joeper214** [comment]
did you tried?

`cd frontend`
`gulp ts`

**Comment by root-beer** [comment]
I tried to specify the version of browserify-rails to 1.5.0, it went well.

**Comment by root-beer** [comment]
I don't have the way to make a test succeed.
so I disable karma test.

**Comment by Joeper214** [comment]
Thank you. Sorry I respond slow because my PC was broken. Good thing that it's fixed now.

**Comment by root-beer** [comment]
:+1:

**Pull request merged**